src/NzbDrone.Core.Test/ImportListTests/ImportListSyncServiceFixture.cs

```diff
@@ -0,0 +1,136 @@
+using System.Linq;
+using System.Collections.Generic;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Core.ImportLists;
+using NzbDrone.Core.MetadataSource;
+using NzbDrone.Core.Tv;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Test.Framework;
+using NzbDrone.Core.ImportLists.Exclusions;
+
+namespace NzbDrone.Core.Test.ImportListTests
+{
+    public class ImportListSyncServiceFixture : CoreTest<ImportListSyncService>
+    {
+        private List<ImportListItemInfo> _importListReports;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var importListItem1 = new ImportListItemInfo
+            {
+                Title = "Breaking Bad"
+            };
+
+            _importListReports = new List<ImportListItemInfo>{importListItem1};
+
+            Mocker.GetMock<IFetchAndParseImportList>()
```

Comment:
Nit: Align on `.`

ReSharper can do it automatically too:
![image](https://user-images.githubusercontent.com/413544/90341157-0805ec00-dfb2-11ea-885f-98ac5cfded4a.png)

Reply:
I don’t use resharper, but we have stylecop for backend on Radarr/Lidarr side. I need to see if there’s a rule to enforce.

On another note, would you guys be open to pulling over stylecop to enforce backend formatting? It’s nice.